cmake.sh

```
@@ -8,8 +8,8 @@ build_requires:
 prefer_system: .*
 prefer_system_check: |
     REQUESTED_VERSION=3.20.0
-    verge() { [[  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]]; }
-    type cmake && verge $REQUESTED_VERSION `cmake --version | sed -e 's/.* //' | cut -d. -f1,2,3`
+    verge() { [[  "$1" = "$(echo -e "$1\n$2" | sort -V | head -n1)" ]]; }
+    type cmake && verge $REQUESTED_VERSION "$(cmake --version | sed -e 's/.* //' | cut -d. -f1,2,3)"
```

Shellcheck (0.11.0)

[info] REQUESTED_VERSION was modified in a subshell. That change might be lost.

(SC2031)